### PR TITLE
Expose Shiki and Remark

### PR DESCRIPTION
**Pull request by mrjvs**
changes:
 - Fix mistakes and clarify some stuff in config documentation
 - Add Shiki and Remark options to `guider()` entrypoint.
 - Update Guider documentation to reflect the new changes
 - Remove dead projects from showcases
 - Bump versions, so it can immediately be released

### CODE REVIEW

File: apps/docs/pages/showcase.tsx [hunk 1]
```diff
@@ -15,20 +13,6 @@ const showcases: ShowcaseType[] = [
     imageUrl: pretendoImg.src,
     tags: ['guider'],
   },
-  {
-    title: 'movie-web backend',
```
Comment (binaryoverload) [inline apps/docs/pages/showcase.tsx:19]: Rip

File: apps/docs/package.json [hunk 2]
```diff
@@ -6,7 +6,8 @@
   "eslintConfig": {
     "extends": [
       "@repo/eslint-config"
-    ]
+    ],
+    "ignorePatterns": ["out/**/*", ".next/**/*"]
```
Comment (binaryoverload) [inline apps/docs/package.json:10]: Node modules maybe?

**Review by binaryoverload — CHANGES_REQUESTED** [review]

File: apps/docs/package.json (hunk 2 above)
Comment (binaryoverload) [reply apps/docs/package.json:10]: Although why ignore these if you've changed the linting to run on the src folder?

Comment (mrjvs) [reply apps/docs/package.json:10]: Src folder was only for `create-guider` project

Comment (mrjvs) [reply apps/docs/package.json:10]: Also node_modules are ignore by default by eslint

**Review by binaryoverload — APPROVED** [review]
Fix your linting smh

**Pull request merged**